pkg/urbit/compat/mingw/compat.c

```diff
@@ -144,6 +144,63 @@ int link(const char *path1, const char *path2)
   return -1;
 }
 
+ssize_t pread(int fd, void *buf, size_t count, off_t offset)
+{
+  DWORD len = 0;
+
+  OVERLAPPED overlapped = {0};
+
+  overlapped.OffsetHigh = (sizeof(off_t) <= sizeof(DWORD)) ?
+                          (DWORD)0 : (DWORD)((offset >> 32) & 0xFFFFFFFFL);
+  overlapped.Offset     = (sizeof(off_t) <= sizeof(DWORD)) ?
+                          (DWORD)offset : (DWORD)(offset & 0xFFFFFFFFL);
```

Comment:
Why not use a macro to avoid repeating `0xFFFFFFFFL` multiple times?
```c
#define DWORD_MASK 0xFFFFFFFFL
```

Reply:
This code is entirely consistent with the conventions used throughout `compat.c`. There are many things about that file that I would change if I was going to refactor it, but all that I'm doing here is adding these two wrapper functions.